File: Logica_Programacao/JAVA/src/faccat/Exer07.java
package faccat;
import java.util.*;

public class Exer07 {
	public static void main (String args[]) {
		Scanner sc = new Scanner(System.in);
		int dia,mes,ano,conta;
		
		System.out.println("Informe quantos anos possui: ");
		ano = sc.nextInt();
		System.out.println("Informe quantos meses de vida tem: ");
		mes = sc.nextInt();
		System.out.println("Informe o dia dos mes: ");
		dia = sc.nextInt();
		
		conta = dia + (mes*30) + (ano * 365);
		
		System.out.println("Os seus dias de vida são: "+conta);
		
		sc.close();
		
	}

}

File: Logica_Programacao/HTML/JS/exer07.js
function exer07() {
    var ano = prompt("Insira quantos anos possui: ");
    var mes = prompt("Insitra quantos meses possui: ");
    var dia = prompt("Insira quantos dias possui: ");
    ano = ano*365;
    mes = mes*30;
    var idade = ano + mes + dia;
     alert("Seus dias vividos são: "+idade);
}